docs-en/chapter_data_structure/basic_data_types.md

```diff
@@ -1,24 +1,24 @@
-# Fundamental Data Types
+# Basic Data Types
 
-When we think of data in computers, we imagine various forms like text, images, videos, voice, 3D models, etc. Despite their different organizational forms, they are all composed of various fundamental data types.
+When discussing data in computers, various forms like text, images, videos, voice and 3D models comes to mind. Despite their different organizational forms, they are all composed of various basic data types.
 
-**Fundamental data types are those that the CPU can directly operate on** and are directly used in algorithms, mainly including the following.
+**Basic data types are those that the CPU can directly operate on** and are directly used in algorithms, mainly including the following.
 
 - Integer types: `byte`, `short`, `int`, `long`.
 - Floating-point types: `float`, `double`, used to represent decimals.
 - Character type: `char`, used to represent letters, punctuation, and even emojis in various languages.
-- Boolean type: `bool`, used for "yes" or "no" decisions.
+- Boolean type: `bool`, used ro represent "yes" or "no" decisions.
 
-**Fundamental data types are stored in computers in binary form**. One binary digit is equal to 1 bit. In most modern operating systems, 1 byte consists of 8 bits.
+**Basic data types are stored in computers in binary form**. One binary digit is 1 bit. In most modern operating systems, 1 byte consists of 8 bits.
```

Comment:
"One binary digit is 1 bit." can be changed to ->
A single binary digit is equivalent to 1 bit.